### PR TITLE
Feature/file upload

### PR DESCRIPTION
**Pull request by Asergey91**
further work on attachments

### CODE REVIEW

**Comment by nvdk** [comment]
@lagartoverde : some missing semi's in app/services/document-service.js

**Comment by lagartoverde** [comment]
Oh, I didn't check the semis yet, this is not ready, the function to get the decisions only runs when opening the modal, I will solve that next now that I finished the prepublisher

**Comment by lagartoverde** [comment]
Ready for review now :)

**Review by abeforgit — COMMENTED** [review]
Looks ok from what I understand of it. Seems to work as well.

May or may not be an issue:
when adding attachments from the draft-decision screen, the "besluit" select box is always empty, even when it does show a value in the zitting view.

File: app/controllers/agendapoints/edit.js [hunk 1]
```diff
@@ -13,6 +13,9 @@ export default class AgendapointsEditController extends Controller {
   @tracked displayDeleteModal = false;
   @tracked _editorDocument;
   profile = 'draftDecisionsProfile';
+  @tracked uploading=false;
```
Comment (nvdk) [inline app/controllers/agendapoints/edit.js:16]: please mind the formatting

File: app/controllers/meetings/edit/treatment.js [hunk 2]
```diff
@@ -14,6 +14,9 @@ export default class MeetingsEditTreatmentController extends Controller {
   @tracked documentContainer;
   @tracked document;
   @tracked initialContent;
+  @tracked uploading=false;
```
Comment (nvdk) [inline app/controllers/meetings/edit/treatment.js:17]: formatting

File: app/services/document-service.js [hunk 3]
```diff
@@ -32,4 +32,24 @@ export default class DocumentService extends Service {
     }
     return prefixesString;
   }
+  getDecisions(editorDocument) {
+    const node = document.createElement('body');
```
Comment (nvdk) [inline app/services/document-service.js:36]: this shares quite a bit of logic with getDescription which can be abstracted

**Review by nvdk — CHANGES_REQUESTED** [review]
- Some small remarks on formatting
- I'd recommend abstracting the extractDecisions part that's present in both getDescription and getDecisions in the document service
- attachment type "regelgevend" is only possible for an attachment linked to a decision

**Review by nvdk — APPROVED** [review]

**Pull request merged**